src/app/globals.css

```
@@ -0,0 +1,149 @@
+  --shadow-sm-x: 0px;
+  --shadow-sm-y: 0px;
+  --shadow-md-x: 1px;
+  --shadow-md-y: 1px;
+  --shadow-lg-x: 3px;
+  --shadow-lg-y: 3px;
+
+  /* Typography */
+  --base-font-weight: 500;
+  --heading-font-weight: 900;
+  --mono-font-family:
+    ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New",
```

Stylelint (17.4.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)